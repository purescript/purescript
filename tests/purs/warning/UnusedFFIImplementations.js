export var yes = true;
export var no = false;
