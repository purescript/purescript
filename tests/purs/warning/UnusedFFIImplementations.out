Warning found:
at tests/purs/warning/UnusedFFIImplementations.purs:2:1 - 4:30 (line 2, column 1 - line 4, column 30)

  The following definitions in the foreign module for module [33mMain[0m are unused:

    no



See https://github.com/purescript/documentation/blob/master/errors/UnusedFFIImplementations.md for more information,
or to contribute content related to this warning.

